### PR TITLE
ECF-RP-335: start page and service name

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
Two small changes here: start page copy, and we now have a service name 🎉 

### Guidance to review
I haven't updated the service name in notify yet, so there's still a couple of references to that. Apart from that, let me know if you think I've missed anything

### CODE REVIEW

**Review by kicferk1 — APPROVED** [review]
There are a few more references
1. `TWD Early Careers Framework` in `school_not_confirmed/show`
2. Lots of `early-careers-framework` in our dev ops. I don't think we want to change all of it, maybe not even some of it, just thought I would mention it in case you disagree :)

**Comment by ethanmills** [comment]
> Lots of early-careers-framework in our dev ops. I don't think we want to change all of it, maybe not even some of it, just thought I would mention it in case you disagree :)

That's a more difficult, but less immediate issue. I decided I didn't want to deal with that today

**Comment by ethanmills** [comment]
> TWD Early Careers Framework in school_not_confirmed/show

That's the notify thing I mentioned. That's still true at the moment

File: app/views/cookies/show.html.erb [hunk 1]
```diff
@@ -18,10 +18,10 @@
 
     <h1 class="govuk-heading-l">Cookies</h1>
     <p>Cookies are small files saved on your phone, tablet or computer when you visit a website.</p>
-    <p>We use cookies to make the Early Career Framework (ECF) work and collect information about how you use our service.</p>
+    <p>We use cookies to make the Manage training for early career teachers service (the service) work and collect information about how you use our service.</p>
```
Comment (callumacrae) [inline app/views/cookies/show.html.erb:21]: Is MTFECTS not catchy enough?

**Review by callumacrae — APPROVED** [review]

**Review by seb-siem-softwire — APPROVED** [review]

**Pull request merged**